### PR TITLE
Add support for WNT 3.0

### PR DESCRIPTION
**Pull request by teemupiiroinenwirepas**
Signed-off-by: Teemu Piiroinen <teemu.piiroinen@wirepas.com>

### CODE REVIEW

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ wrappers/python/tests/wnt/utils.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ wrappers/python/tests/wnt/utils.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ wrappers/python/tests/wnt/utils.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

File: wnt/protocol_buffers_files/commons.proto [hunk 1]
```diff
@@ -26,8 +26,15 @@ message FullRole {
...
+    enum Type {
+        NETWORK = 1;
+        SINK = 2;
+    }
+
     optional uint32 interval = 1;  // Diagnostics interval in seconds
     optional uint32 sequence = 2;
     optional bytes app_config = 3 [(nanopb).max_size = 1024];
     optional uint32 max_length = 4; // Maximum length of the app config
+    optional bool is_override_on = 5; // WNT backend will override appconfig it it is changed outside of WNT
```
Comment (GwendalRaoul) [inline wnt/protocol_buffers_files/commons.proto]: Nit: typo: "if it is" ;-)

Comment (teemupiiroinenwirepas) [reply wnt/protocol_buffers_files/commons.proto]: Yeah. I'll fix

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ wrappers/python/tests/wnt/utils.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ wrappers/python/tests/wnt/utils.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ wrappers/python/tests/wnt/utils.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ wrappers/python/tests/wnt/utils.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ wrappers/python/tests/wnt/utils.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Comment by PFigs** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Clones removed
==============
+ wrappers/python/tests/wnt/utils.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/wirepas/backend-apis/pullRequest?prid=4949339&bid=16359753)

**Pull request merged**